### ISSUE TITLE
Documentation improvements

### ISSUE THREAD

**Issue by narFnarF** [TheTechnobear/Orac#13]
As far as I can tell, the wiki/documentation is missing important parts. Some parts are covered in videos, some is only covered in forums and issue tickets, etc. I think the wiki here would be a great place to centralize the information, and provide it in written format, much easier to maintain than videos.

As a new user, I struggled to install Orac, setup it and make it work.

Here are the parts that I think could be improved:

- More detailed install instructions for Raspberry Pi
- Where to get the pd Remore Control patches and how to make it work
- Installing and setuping patchbox (just provide links to their documentation)
- Troubleshoot list
- Probably other things that I can't think of right now

Basically, I think we should have a guide for new users that help them setup the whole thing.

I'm not too bad at writing documentation, so I could help by writing a first draft for each, as long as someone is willing to review it because I'm new to Orac and still fairly new to linux and raspberry pi in general. 

Some of these infos could easily be added to existing wiki pages, however, I'm hesitant to make the changes myself without someone reviewing it.

What do you think?

**Comment by Alberaan** [comment]
As a new user too, I found it difficult to find information on how to map the Aux Button in my midi controller for the raspberry pi system. I've just edited the FAQ page to include the answer to this question. Also, I'm having trouble to find exact instructions on how to map controls for changing the modules loaded in the patch as well as navigating over the active module using a midi controller. I've read somewhere in a forum that this can be done with program change messages, but I haven't been able to make it work yet. Will need to research more on this topic. When I do, I will also suggest an edit on the FAQ page.

**Comment by TheTechnobear** [comment]
Orac is an open source project that I develop for free in my own free time.
as you may imagine, development and support takes a lot of time... 
so as much as I would like to , I do not have time to devote to improving the documentation.

but as its open source, the community of orac users could contribute to this documentation effort.
new users are very placed to participate in this... since they are best placed to see what is missing.
(developers notoriously cannot see whats difficult to understand, since we are too close to the project !)

the questions asked here, I have answered many times.. 
on youtube comments (on my videos) ,  and on forums.
the best places to search for these answers are : 

https://community.blokas.io/t/orac-2-0-for-the-raspberry-pi/1099
https://forum.critterandguitari.com/t/orac-2-0/4154/81

(also search for orac, and what you are missing on these forums, as there are other threads)

I would **really** appreciate contributions in this area.

as you may notice, I personally create/shared and contribute to many open source projects.
in the hope that others can benefits from my efforts... but I only have so much free time. ( I do like to make music myself as well ;) ) 

also... whilst I know new users who are stuck will disagree, but the majority of users would probably prefer I use my time for improving (technically/developing) projects, rather than only have time for support and documentation.
I know... a lame developer excuse perhaps... but when you do things like this in your free time, you have to compromise :(

**Comment by Alberaan** [comment]
Hello Mark!

First of all, thanks for all of your work and the time you spent writting the reply.

I am willing to contribute to improve documentation, and as I've said in my previous message, I've already edited the FAQs in the wiki to include an answer I found elsewhere. I'm also willing to keep on adding these kind of improvements in the future. Is this the correct way to improve the documentation?

**Comment by TheTechnobear** [comment]
oops, sorry missed this..

thank you for your contribution @Alberaan 
yes, I think editing the wiki to add useful information is definitely the best way :)

**Comment by Alberaan** [comment]
@TheTechnobear no problem man. The only thing I'm missing to complete my setup is the right midi messages to be able to navigate in the menus. So, when clicking on the arrows in the ORAC client, I get to select different options. I want to be able to do this directly with my midi device instead of relying on a pc or a mobile client (I got the ORAC client running on Android perfectly btw) and maybe I can use Android just as a screen.
